deployment/deploy/2010-deploy-ReplaceLoanTokenModules.js

```diff
@@ -0,0 +1,149 @@
+const path = require("path");
+const { getContractNameFromScriptFileName } = require("../helpers/utils");
+const hre = require("hardhat");
+const { sendWithMultisig } = require("../helpers/helpers");
+const col = require("cli-color");
+
+const { arrayToUnique } = require("../helpers/utils");
+
+const func = async function (hre) {
```

Comment:
pls add check & skip registration of already registered modules - those which deployment has not changed since the ast registration.  if the contract doesn't change, hh will just reuse it and won't deploy.

Reply:
Done